prow/manifests/overlays/metal3/config.yaml

```diff
@@ -1487,6 +1494,9 @@ presubmits:
 
   metal3-io/ironic-image:
     - name: shellcheck
+      branches:
```

Comment:
This is not necessary. Shellcheck should be ran for all branches, it has always been there.

Markdownlint we replaced between 23.1 and 24.0, so it differs as we did not backport the patches.

Reply:
weird copy-paste here, I removed it